app/mailers/app_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class AppMailer < ActionMailer::Base
+  def welcome(user)
+    @user = user
+    mail from: 'info@myflix.com', to: user.email, subject: "You are now a member of the MyFlix Family!"
```

Comment:
You can set the repetitive stuff like `from` as a default, then you don't have to repeat it.